source/manual/alerts/publisher-unprocessed-fact-check-emails.html.md

```diff
@@ -0,0 +1,44 @@
+---
+owner_slack: "#govuk-2ndline"
+title: 'Publisher: Unprocessed fact-check emails'
+section: Icinga alerts
+subsection: Email alerts
+layout: manual_layout
+parent: "/manual.html"
+last_reviewed_on: 2020-07-02
+review_in: 6 months
+---
+
+### Unprocessed fact-check emails
+
+This means there are unprocessed emails in the fact-check inbox
+(govuk-fact-check@digital.cabinet-office.gov.uk).
+Publisher has a [script (`mail_fetcher`)](script) that checks the inbox
+[every 5 minutes][schedule] and [processes all emails present][process].
+
+If emails are not processed or deleted, this alert is triggered. The level of
+the [alert][] depends on the number of emails present - which was chosen
+arbitrarily and is currently:
+- warning with 1 email
+- critical with 2 or more emails
+
+### Log into the fact-check inbox
+
+If this alert is triggered, we want to log into the fact-check inbox and
+investigate why the emails were not processed and deleted.
+
+The password to log into the govuk-fact-check@digital.cabinet-office.gov.uk
+inbox can be found by running:
+
+```bash
+PASSWORD_STORE_DIR=~/govuk/govuk-secrets/pass/2ndline pass google-accounts/govuk-fact-check@digital.cabinet-office.gov.uk
+```
+
+The same can be done for
+staging (govuk-fact-check-staging@digital.cabinet-office.gov.uk)
+and integration (govuk-fact-check-integration@digital.cabinet-office.gov.uk).
+
+[script]: https://github.com/alphagov/publisher/blob/master/script/mail_fetcher
+[schedule]: https://github.com/alphagov/publisher/blob/master/config/schedule.rb#L7-L9
+[process]: https://github.com/alphagov/publisher/blob/master/lib/fact_check_email_handler.rb#L41-L53
```

Comment:
Might be an idea for this link and the one above it to reference a blob rather than `master`, as the line positions can change.